web/viewer.html

```diff
@@ -142,10 +142,14 @@
               <span data-l10n-id="find_next_label">Next</span>
             </button>
           </div>
-          <input type="checkbox" id="findHighlightAll" class="toolbarField" tabindex="94">
-          <label for="findHighlightAll" class="toolbarLabel" data-l10n-id="find_highlight">Highlight all</label>
-          <input type="checkbox" id="findMatchCase" class="toolbarField" tabindex="95">
-          <label for="findMatchCase" class="toolbarLabel" data-l10n-id="find_match_case_label">Match case</label>
+          <label data-l10n-id="find_highlight">
```

Comment:
The content of the element identified by `data-l10n-id="find_highlight"` will be replaced with the translation. So the checkbox will disappear with this change.

You should remove the attribute from here, wrap "Highlight all" in a `<span>` and put the attribute on the `<span>`. Same for the other checkboxs.